src/Categories/Adjoint/Parametric.agda

```diff
@@ -0,0 +1,150 @@
+module Categories.Adjoint.Parametric where
+
+open import Level
+
+open import Data.Product
```

Comment:
Adding some extra `using` below would be appreciated.

Reply:
Done, also cleaned up some unused imports.